docs/content/crds/templates/markdown.tmpl

```diff
@@ -0,0 +1,102 @@
+# API Reference
```

Comment:
This is just a copy/paste of the default template used by crdoc. however, I figured it would be easier for developers to change if we placed it in the repository directly

Reply:
Does that mean we might have drift? But that we want updates to be easy and under our control and so we're relying on our own template and not theirs going forward? Just want to make sure I understand.

Reply:
There's not "drift" per se. We have a single format in our docs (that didn't exist 2 weeks ago before we added this). That format was inherited as the default from another repo, but we will likely want to update it. So the real source of truth is the one we're adding, and if it "drifts" from the default of the source repo, that is ok (and likely expected)